docs/gettingstarted/modfiles.md

```diff
@@ -248,3 +248,39 @@ An entry in `neoforge.mods.toml` does not need a corresponding `@Mod` annotation
 [versioning]: versioning.md
 [enumextension]: ../advanced/extensibleenums.md
 [featureflags]: ../advanced/featureflags.md
+## Mod files tree
+```
+root
+ |-- .gradle
+ |-- gradle
+ |-- build
+ |-- run
+ |-- src
+ |    |-- main
+ |    |    |-- java
+ |    |    |    |-- <com/net>
+ |    |    |    |    |-- <username>
+ |    |    |    |    |    |-- <modid>
+ |    |    |    |    |    |    |-- <some packages and single files>
+ |    |    |    |    |    |    |-- <modname>.java
+ |    |    |    |    |    |    |-- config.java
+ |    |    |-- resources
+ |    |    |    |-- assets
+ |    |    |    |    |-- <modid>
+ |    |    |    |-- data
+ |    |    |    |    |-- minecraft
+ |    |    |    |    |-- <modid>
+ |    |-- test
+ |-- build.gradle
+ |-- gradle.properties
+ |-- gradlew   (file)
+ |-- gradlew.bat
+ |-- settings.gradle
+```
+The `gradle` and `.gradle` folders are Gradle source folders (of the ModDevGradle plugin) that you will not normally need to modify.
+
+The `run` folder contains all data generated by testing your mod (like the `save` folder with all the world data).
+
+The `src` folder is the center of modding. `test` is optionnal. `java` is the source code and `resources` contains a `àssets` folder, that you've need to build as a [resources pack][https://minecraft.wiki/w/Resource_pack]. `data` is the same as a [datapack][https://minecraft.wiki/w/Data_pack], where <modid> is your mod id (like `examplemod`) and `minecraft` is normally only for build-in tags.
+
+The java folder is always build as `java/<com/net>/<username>/<modid>` where `<com/net>` is a [web domains type][https://en.wikipedia.org/wiki/List_of_Internet_top-level_domains] and `<username>` is your pseudo. Then, in the last child, it's always a `<modid>.java` file, as `ExampleMod.java` and a `config.java` (see [config][config.md]). Packages are also descripted in [structuring.md].
```

Comment:
This is, once again, an assumption of how the user chooses to structure their files. This is why we have a structuring section.